### PR TITLE
[Components with more than 1 service] LPS-183239 Refactor PortetProvider components from Commerce modules part II

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://issues.liferay.com/browse/LPS-183239

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:commerce

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

File: modules/apps/commerce/commerce-shipment-web/src/main/java/com/liferay/commerce/shipment/web/internal/portlet/CommerceShipmentPortletProvider.java [hunk 1]
```diff
@@ -27,14 +25,10 @@
...
-	}
+	service = EditPortletProvider.class
 )
 public class CommerceShipmentPortletProvider
-	extends BasePortletProvider
-	implements EditPortletProvider, ManagePortletProvider, ViewPortletProvider {
+	extends BasePortletProvider implements EditPortletProvider {
 
 	@Override
 	public String getPortletName() {
```
[the remark this answers is not in the document]
Comment (rafaprax) [reply modules/apps/commerce/commerce-shipment-web/src/main/java/com/liferay/commerce/shipment/web/internal/portlet/CommerceShipmentPortletProvider.java:34]: please apply it to other cases. Thx

**Comment by rafaprax** [comment]
Please resend when you finish the requested changes. Thx

**Pull request closed without merging**